### ISSUE TITLE
Tools packages not parsed

### ISSUE THREAD

**Issue by skolima** [praeclarum/FuGetGallery#21]
Example: xunit.console.runner https://fuget.org/packages/xunit.runner.console

Missing the target frameworks detection (I guess because files are in tools\ instead of the usual location). Package contents are listed, but no assembly analysis results are available.

**Comment by praeclarum** [comment]
Yeah tools definitely don't get the same treatment as lib. I will look to unify the code paths.
